docs/src/modules/components/AppNavDrawer.js

```diff
@@ -25,10 +25,10 @@ import DoneRounded from '@mui/icons-material/DoneRounded';
 import FEATURE_TOGGLE from 'docs/src/featureToggle';
 import { isNewLocation } from 'docs/src/modules/utils/replaceUrl';
 import MuiProductSelector from 'docs/src/modules/components/MuiProductSelector';
-import materialPkgJson from '../../../../packages/mui-material/package.json';
-import joyPkgJson from '../../../../packages/mui-joy/package.json';
-import basePkgJson from '../../../../packages/mui-base/package.json';
-import systemPkgJson from '../../../../packages/mui-system/package.json';
+import materialPkgJson from 'packages/mui-material/package.json';
```

Comment:
This doesn't work in MUI X inside `@mui/monorepo` package.
I'll submit a PR to revert this as a quick fix

Reply:
Ohh, now we have solved the mystery! cc @Janpot

> For some reason MUIX setup understands the packages/... alias and I can't figure out why.

https://github.com/mui/mui-toolpad/pull/542#issuecomment-1175186429

It's simply because we were not looking at the right source. MUI X didn't update to the latest version of HEAD yet.